src/common/queries/tasks.ts

```diff
@@ -40,25 +40,21 @@ export function useBlankTaskQuery() {
 }
 
 interface TasksParams {
-  options?: {
-    limit: number;
```

Comment:
Me and @turbo124 discussed on the slack and realized that the `limit` option had no use here so it was removed. Instead, `per_page` will handle this part about limiting the results.